<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewProjectDialog</class>
 <widget class="QDialog" name="NewProjectDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>511</width>
    <height>290</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>项目属性</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>6</number>
   </property>
   <property name="topMargin">
    <number>6</number>
   </property>
   <property name="rightMargin">
    <number>6</number>
   </property>
   <property name="bottomMargin">
    <number>6</number>
   </property>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>项目名</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="ProjectNameEdit"/>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>项目路径</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLineEdit" name="ProjectPathEdit"/>
     </item>
     <item>
      <widget class="QPushButton" name="BrowseButton">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>描述</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="CommentEdit"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="OkButton">
       <property name="text">
        <string>确定</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="CancelButton">
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <zorder>label</zorder>
  <zorder>ProjectPathEdit</zorder>
  <zorder>label_2</zorder>
  <zorder>BrowseButton</zorder>
  <zorder>CommentEdit</zorder>
  <zorder>ProjectNameEdit</zorder>
  <zorder>OkButton</zorder>
  <zorder>CancelButton</zorder>
  <zorder>label_3</zorder>
  <zorder>horizontalSpacer</zorder>
 </widget>
 <tabstops>
  <tabstop>ProjectNameEdit</tabstop>
  <tabstop>ProjectPathEdit</tabstop>
  <tabstop>BrowseButton</tabstop>
  <tabstop>CommentEdit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
